src/drive/web/modules/actions/index.jsx

```diff
@@ -67,7 +67,8 @@ export const trash = ({
   pushModal,
   popModal,
   hasWriteAccess,
-  refresh
+  refresh,
+  refreshFolderContent
```

Comment:
Why is the refreshFolderContent not done in the `refresh` callback passed from above ?